docs/pages/reference/mesheryctl/mesheryctl-app-view.md

```diff
@@ -76,7 +76,7 @@ mesheryctl app view --all
 ## Screenshots
 
 Usage of mesheryctl app view
-![app-view-usage](../../../../docs/assets/img/mesheryctl/app-view.png)
+![app-view-usage](https://docs.meshery.io/assets/img/mesheryctl/app-view.png)
```

Comment:
@alphaX86, can you update the image link to be relative in other image refs (like this one) too.

Reply:
Got it, updating now